app/views/catalog/_members_ephemera_project.html.erb

```diff
@@ -1,9 +1,32 @@
   <h2>Boxes</h2>
-  <ul>
+  <table class="table table-striped datatable">
+    <thead>
+    <tr>
+      <th>Workflow State</th>
+      <th>Title</th>
+      <th>Barcode</th>
+      <th>Actions</th>
+    </tr>
+    </thead>
+    <tbody>
     <% decorated_resource.boxes.each do |box| %>
-      <li><%= link_to "Box #{box.box_number.first}", Valhalla::ContextualPath.new(child: box, parent_id: nil).show %></li>
+      <% box_url = parent_solr_document_path("#{resource.id}", "#{box.id}") %>
```

Comment:
👍 , noting this improvement for cases where `EphemeraFolders` may also be rendered as `EphemeraBoxes` are.